doc/api/readline.md

```diff
@@ -373,6 +373,11 @@ changes:
   * `removeHistoryDuplicates` {boolean} If `true`, when a new input line added
     to the history list duplicates an older one, this removes the older line
     from the list. **Default:** `false`.
+  * `escapeCodeTimeout` {number} The duration `readline` will wait for a
+    character when reading an ambiguous key sequence (one that can both form a
+    complete key sequence using the input read so far and can take additional
+    input to complete a longer key sequence). The default value is used if
+    `NaN`, a negative number, or a non-number is provided. **Default:** `500`.
```

Comment:
It is unclear in what unit the time is measured here. It should say something like `The duration readline will wait for a character when reading an ambiguous key sequence in milliseconds (...)`.

I personally would also only accept integers but that is just me.